pkg/networkserver/grpc_gsns.go

```diff
@@ -1274,3 +1274,9 @@ func (ns *NetworkServer) HandleUplink(ctx context.Context, up *ttnpb.UplinkMessa
 	logger.Debug("Unmatched MType")
 	return ttnpb.Empty, nil
 }
+
+// ReportTxAcknowledgment is called by the Gateway Server when a tx acknowledgment is received.
+func (ns *NetworkServer) ReportTxAcknowledgment(ctx context.Context, up *ttnpb.GatewayTxAcknowledgment) (_ *pbtypes.Empty, err error) {
+	// TODO: Forward TxAcknowledgment to the Application Server (https://github.com/TheThingsNetwork/lorawan-stack/issues/76)
```

Comment:
@rvolosatovs this is probably a simple call, can you chime in here?

Reply:
This is part of #56 instead of #76, so let's do in a separate PR.

Reply:
Then update the issue number here